<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profile</class>
 <widget class="QWidget" name="Profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1671</width>
    <height>901</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile</string>
  </property>
  <widget class="QPushButton" name="nextPostButton">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>700</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

background-color:  rgb(30, 101, 255);
color:rgb(255, 255, 255);
border-radius: 15px;

}
QPushButton:hover{
background-color: rgb(1, 221, 255);
color:rgb(0, 0, 0);
}
QPushButton:pressed{
background-color:rgb(43, 107, 255)
}</string>
   </property>
   <property name="text">
    <string>Next post</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previousPostButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>700</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

background-color:  rgb(30, 101, 255);
color:rgb(255, 255, 255);
border-radius: 15px;

}
QPushButton:hover{
background-color: rgb(1, 221, 255);
color:rgb(0, 0, 0);
}
QPushButton:pressed{
background-color:rgb(43, 107, 255)
}</string>
   </property>
   <property name="text">
    <string>Prev. post</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="postArea">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>751</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(207, 217, 255);
</string>
   </property>
   <property name="title">
    <string>posted on -Date-</string>
   </property>
   <widget class="QGroupBox" name="commentsBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>601</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Comments</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="postTextBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>711</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="commentLine">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>530</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="postCommentButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>530</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

background-color:  rgb(30, 101, 255);
color:rgb(255, 255, 255);
border-radius: 15px;

}
QPushButton:hover{
background-color: rgb(1, 221, 255);
color:rgb(0, 0, 0);
}
QPushButton:pressed{
background-color:rgb(43, 107, 255)
}</string>
    </property>
    <property name="text">
     <string>Post comment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="likeButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>290</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

background-color:  rgb(30, 101, 255);
color:rgb(255, 255, 255);
border-radius: 15px;

}
QPushButton:hover{
background-color: rgb(1, 221, 255);
color:rgb(0, 0, 0);
}
QPushButton:pressed{
background-color:rgb(43, 107, 255)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/2x/baseline_thumb_up_white_24dp.png</normaloff>Icons/2x/baseline_thumb_up_white_24dp.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="dislikeButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>290</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

background-color:  rgb(30, 101, 255);
color:rgb(255, 255, 255);
border-radius: 15px;

}
QPushButton:hover{
background-color: rgb(1, 221, 255);
color:rgb(0, 0, 0);
}
QPushButton:pressed{
background-color:rgb(43, 107, 255)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/2x/baseline_thumb_down_alt_white_24dp.png</normaloff>Icons/2x/baseline_thumb_down_alt_white_24dp.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="referenceUsername">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>691</width>
      <height>181</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="referenceTweetText">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>631</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="char_counter">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>570</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QScrollArea" name="commentsScrollArea_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>691</width>
      <height>141</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="commentContentScrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>689</width>
       <height>139</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="nameInitLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
 border-style: outset;
 border-width: 2px;
 border-color: black;
border-radius:25px;
font: 20pt &quot;Lucida Calligraphy&quot;;
color:black;
background-color:rgb(120, 149, 255)
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="usernamePostLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 13pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="followButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

background-color:  rgb(30, 101, 255);
color:rgb(255, 255, 255);
border-radius: 15px;

}
QPushButton:hover{
background-color: rgb(1, 221, 255);
color:rgb(0, 0, 0);
}
QPushButton:pressed{
background-color:rgb(43, 107, 255)
}</string>
   </property>
   <property name="text">
    <string>Follow</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1671</width>
     <height>901</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(213, 221, 255)
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>110</y>
     <width>630</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(185, 200, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="feedButton">
   <property name="geometry">
    <rect>
     <x>1450</x>
     <y>50</y>
     <width>134</width>
     <height>39</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 10px;
background-color:rgb(53, 141, 255);
color:rgb(255, 255, 255)
}
QPushButton:hover{
background-color:rgb(33, 81, 255)
}
QPushButton:pressed{
background-color:rgb(43, 107, 255)
}</string>
   </property>
   <property name="text">
    <string>Feed</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Icons/2x/baseline_home_black_24dp.png</normaloff>Icons/2x/baseline_home_black_24dp.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>label_2</zorder>
  <zorder>nextPostButton</zorder>
  <zorder>previousPostButton</zorder>
  <zorder>nameInitLabel</zorder>
  <zorder>usernamePostLabel</zorder>
  <zorder>followButton</zorder>
  <zorder>postArea</zorder>
  <zorder>feedButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
